React/CoreModules/RCTDevSettings.mm

```diff
@@ -241,6 +247,8 @@ - (BOOL)isNuclideDebuggingAvailable
 // [TODO(OSS Candidate ISS#2710739)
 RCT_EXPORT_METHOD(setDevModeEnabled:(BOOL)enabled)
 {
+#undef RCT_DEV
+#define RCT_DEV enabled // redfine the macro at runtime
```

Comment:
This won't actually have any impact, will it? Only affects preprocessor, which is irrelevant after preprocessing phase before even compiling